develop/dev-guide-aws-appflow-integration.md

```diff
@@ -0,0 +1,248 @@
+
+```sql
+test> SELECT * FROM sf_account;
++--------------------+-------------------------------------+--------------------+---------------+--------+----------------+
+| id                 | name                                | type               | billing_state | rating | industry       |
++--------------------+-------------------------------------+--------------------+---------------+--------+----------------+
+| 001Do000003EDTlIAO | Sample Account for Entitlements     | null               | null          | null   | null           |
+| 001Do000003EDTZIA4 | Edge Communications                 | Customer - Direct  | TX            | Hot    | Electronics    |
+| 001Do000003EDTaIAO | Burlington Textiles Corp of America | Customer - Direct  | NC            | Warm   | Apparel        |
+| 001Do000003EDTbIAO | Pyramid Construction Inc.           | Customer - Channel | null          | null   | Construction   |
+| 001Do000003EDTcIAO | Dickenson plc                       | Customer - Channel | KS            | null   | Consulting     |
+| 001Do000003EDTdIAO | Grand Hotels & Resorts Ltd          | Customer - Direct  | IL            | Warm   | Hospitality    |
+| 001Do000003EDTeIAO | United Oil & Gas Corp.              | Customer - Direct  | NY            | Hot    | Energy         |
+| 001Do000003EDTfIAO | Express Logistics and Transport     | Customer - Channel | OR            | Cold   | Transportation |
+| 001Do000003EDTgIAO | University of Arizona               | Customer - Direct  | AZ            | Warm   | Education      |
+| 001Do000003EDThIAO | United Oil & Gas, UK                | Customer - Direct  | UK            | null   | Energy         |
+| 001Do000003EDTiIAO | United Oil & Gas, Singapore         | Customer - Direct  | Singapore     | null   | Energy         |
+| 001Do000003EDTjIAO | GenePoint                           | Customer - Channel | CA            | Cold   | Biotechnology  |
+| 001Do000003EDTkIAO | sForce                              | null               | CA            | null   | null           |
++--------------------+-------------------------------------+--------------------+---------------+--------+----------------+
+```
+
+## Noteworthy things
+
+- If anything goes wrong, you can navigate to the [CloudWatch](https://console.aws.amazon.com/cloudwatch/home) page on the AWS Management Console to get logs.
+- The steps in this document are based on [Building custom connectors using the Amazon AppFlow Custom Connector SDK](https://aws.amazon.com/blogs/compute/building-custom-connectors-using-the-amazon-appflow-custom-connector-sdk/).
+- [TiDB Cloud Serverless Tier](https://docs.pingcap.com/tidbcloud/select-cluster-tier#serverless-tier-beta) is **NOT** a production environment.
+- To prevent excessive length, the examples in this document only show the `Insert` strategy, but `Update` and `Upsert` strategies are also tested and can be used.
```

Comment:
Please delete the following images, which are not used in the doc:

- media/develop/aws-appflow-step-auth.png
- media/develop/aws-appflow-step-sam-putput.png

Reply:
Done